jpype/reflect.py

```diff
@@ -16,31 +16,33 @@
 #*****************************************************************************
 import _jpype
 
+# FIXME this class is entirely replaced with calls to .class_ and should likely be deprecated.

```

Comment:
This undocumented file should likely be deprecated.  All functionality is in .class_